src/Tags.zig

```diff
@@ -44,17 +44,23 @@ pub fn findTags(self: *Tags, fname: []const u8) anyerror!void {
         var file = try std.fs.cwd().openFile(fname, .{});
         defer file.close();
 
-        const size = try file.getEndPos();
+        const metadata = try file.metadata();
+        const size = metadata.size();
         if (size == 0) {
             return;
         }
+
+        if (metadata.kind() == .Directory) {
+            return error.NotFile;
+        }
+
```

Comment:
Is this necessary? If we don't include this what problems occur?

Reply:
Explained above